src/zif_logger.intf.abap

```diff
@@ -111,4 +111,10 @@ INTERFACE zif_logger
     IMPORTING
       profile TYPE bal_s_prof OPTIONAL.
 
+  METHODS set_desc
```

Comment:
since the other methods aren't abbreviated, i suggest `set_description`. or maybe it could be more generic like `set_header` so we could add other header fields (later).

Reply:
Thx for your feedback. Agree, changed to `SET_HEADER`.